givitsite/coordinate/views.py

```diff
@@ -1,3 +1,131 @@
+        #     "friend_id":"1234",
+        #     "picture": '',
+        #     "request_id": '1',
+        #     "user_id": '2'
+        # })
+
+        # itemsToCoordinate.append({
+        #     "item":'chair',
+        #     "pickup_location":'Jerusalem',
+        #     "drop_off_location":'Tel Aviv',
+        #     "url": 'https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcRxJI6_VVfFWUPcvozhhY7qNVWaujOLj1zI8A&usqp=CAU',
+        #     "friend_id":"1235",
+        #     "picture": '',
+        #     "request_id": '2',
+        #     "user_id": '3'
+        # })
+
+        # itemsToCoordinate.append({
+        #     "item":'chair',
+        #     "pickup_location":'Jerusalem',
+        #     "drop_off_location":'Tel Aviv',
+        #     "url": 'https://ak1.ostkcdn.com/images/products/13223495/TidySquares-Classic-White-Wood-63-Reachin-Closet-Organizer-Design-2-998c7350-e3b4-4cfd-b3eb-67998079718c.jpg',
+        #     "friend_id":"1235",
+        #     "picture": '',
+        #     "request_id": '2',
+        #     "user_id": '3'
+        # })
+
+        return render (request, 'coordinator.html', {'matchedItems':matchedItems, 'openRequests':openRequests, 'users': users})
+
+    elif request.method == 'POST':
+        save_to_coordinations(request);
+        update_friendreq_db(request);
+        return redirect('/coordinate')
+    
+   
+
+def save_to_coordinations(request):
+
+    print("Times:")
+    print(request.POST['pickup_time']);
+    print(request.POST['drop_off_time']);
+
+    request_id = request.POST['request_id']
+
+    if request.method == 'POST':
+        CoordinatedItems.objects.create(
+            friend_id = request.POST['friend_id'],
+            request_id = request.POST['request_id'], 
+            pickup_location = request.POST['pickup_location'],
+            drop_off_location = request.POST['drop_off_location'],
+            item = request.POST['item'],
+            date = request.POST['date'],
+            pickup_time = request.POST['pickup_time'],
+            drop_off_time = request.POST['drop_off_time'],
+            status = 'coordinated'
+            ) 
+    
+
+def update_friendreq_db(request):
+    ItemRequest.objects.filter(id = request.POST['request_id']).update(status='closed')
+
+
+def test(request):
+    print(sys.path)
+    return render (request, 'home.html')
+
+
+def create_file(request):
+    fh = open('brief.txt', 'w+')
+    date_of_transfer = request.POST['date']
+    print(date_of_transfer)
+    fh.write('brief for transfer date - '+ date_of_transfer + '\n\n')
+    itemsForBrief = CoordinatedItems.objects.all(); #filter(date = date_of_transfer)
+    count = 1;
+
+    for item in itemsForBrief: #.objects.filter(date = date_of_transfer):
+        item_title = "item number " + str(count) + '\n'
+        fh.write(item_title)
+        fh.write(
+                    'item: ' + item.item + '\n' +
```

Comment:
use consts for all those parameters (lines 109-113)

Reply:
moved all constants to a new file ('constants') according to a recommendation in this thread: 
https://stackoverflow.com/questions/11111632/python-best-cleanest-way-to-define-constant-lists-or-dictionarys